server/server.js

```diff
@@ -157,6 +158,10 @@ function server(serviceManager, options) {
       case 'express:usage-record':
         ExpressUsageRecord.recordUsage(instanceId, uInfo, callback);
         break;
+      case 'status:wd':
```

Comment:
Not using this notification at all. The data that this provides already comes with other notifications like fork, listening etc.